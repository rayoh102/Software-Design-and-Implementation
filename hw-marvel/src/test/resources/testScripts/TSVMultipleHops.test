# Tests if we can find a shortest path that is multiple hops

# Load Graph
LoadGraph custom sampleData.tsv

# Find path (should be two hops)
FindPath custom Hero1 Hero7