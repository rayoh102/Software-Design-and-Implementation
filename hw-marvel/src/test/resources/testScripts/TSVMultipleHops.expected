# Tests if we can find a shortest path that is multiple hops

# Load Graph
loaded Graph custom

# Find path (should be two hops)
path from BigHero to OtherHero:
Hero1 to Hero4 via B1
Hero4 to Hero8 via C1